modules/serverless-installing-cli-linux-ibm-z_tarball.adoc

```diff
@@ -0,0 +1,45 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="installing-cli-linux_ibm_z_tarball_{context}"]
+
+== Tarball Installation
+
+For Linux distributions, you can download the CLI directly as a `tar.gz` archive.
+
+
+.Procedure
+
+. Download the link:https://mirror.openshift.com/pub/openshift-v4/clients/serverless/latest[CLI].
+
+. Unpack the archive:
++
+
+[source,terminal]
+----
+$ tar -xf <file>
+----
+
+. Move the `kn` binary to a directory on your PATH.
+
+. To check your path, run:
++
+
+[source,terminal]
+----
+$ echo $PATH
+----
+
++
+[NOTE]
+====
+If you do not use RHEL or Fedora, ensure that *libc* is installed in a directory on your library path.
+If *libc* is not available, you might see the following error when you run CLI commands:
+
+[source,terminal]
+----
+$ kn: No such file or directory
+----
+====
```

Comment:
@vikram-redhat @sjstout what do we usually do in OCP docs wrt mentioning things like Fedora or RHEL? It seems sort of odd to me to call it out in a module like this, but maybe not?